portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeStandardHead.vm

```diff
@@ -76,6 +76,8 @@
         <script src="${pageSkinRepo}/${pageSkin}/js/lib/modernizr.js$!{portalCDNQuery}"></script>
         <script src="/library/js/sakai-message-broker.js$!{portalCDNQuery}"></script>
         <script src="/library/js/sakai-lti-advantage.js$!{portalCDNQuery}"></script>
+        <script src="${pageWebjarsPath}vue/2.6.14/vue.min.js$!{portalCDNQuery}"></script>
```

Comment:
if this can be brought via ES6 import

Reply:
Ill add a flag to the vue compiler, then this line is not needed.